_digest/october-2020/making-the-most-out-of-your-morning.mdx

```diff
@@ -0,0 +1,76 @@
+  Scrambling to get ready, you dash out of the house forgetting breakfast and your binders behind. 
+  You certainly don’t have time to go back, otherwise, you would be late. 
+  Taking a seat in frustration, you have nothing to follow along with. 
+  You’re going to need to catch up on the hour’s worth of content tonight but you have extracurriculars planned in the evening. 
+  Feelings of anger, regret, and rage appear throughout the day. 
+  Nothing is going well, it’s a total disaster and on top of that, it’s a Monday! </p>
+  <p> If this scenario sounds familiar, you aren’t alone. 
+  While the term “morning” is an ear-sore for many students, it is one of the most important times of the day. 
+  However, many associate their morning with three simple tasks: waking-up, getting dressed, and breakfast. 
+  Although, that may not want to be the routine you should have if you are a student en route to medical school. 
+  Creating a routine with a sufficient amount of time to do tasks allows you to kick off the day with productivity. 
+  Here are some reasons why morning routines are beneficial. </p>
+ 
+ <div align="center">
+		<img src="/blog/october-2020/alarm-clock-snooze.png" alt="Hand hitting snooze on alarm clock" width="500"/>
+	</div>
+
+<h1>1. Prevents Morning Stress</h1>
+    <p>The feeling of being rushed in the morning leads to distractedness later as the day goes on. 
+    Establishing effective morning habits will enable you to plan out your schedule in an easy manner. 
+    This could be waking up a few hours before your first class to review notes, checking your emails, or going through easy tasks that need to be done. 
+    Checking off these minimal duties will allow your morning to start with productivity and get your brain active.</p>
+<h1>2. Emotions</h1>
+    <p>Being physically happy can influence how we feel emotionally. 
+    When a routine of stress and anxiety is brought upon every morning of your day, one would certainly feel in a state of hopelessness as if you cannot catch up to the rest of your scheduled tasks. 
+    Having a morning routine will start you off feeling good about yourself. 
+    This will certainly bring upon resilience to obstacles needed to be faced for the rest of the day. 
+    Checking off completed tasks certainly brings a sense of pride and accomplishment. </p>
+
+<div align="center">
+		<img src="/blog/october-2020/good-morning-sign.png" alt="Good morning! Let the stress begin sign" width="500"/>
+	</div>
+
+<h1>3. More time for yourself</h1>
+    <p>If you are anything like me, even checking off the tiniest tasks would make me feel very accomplished. 
+    Moreover, it gets tasks out of the way making me feel more motivated and productive. 
+    Completing these minimal tasks frees up time for other work needed to be done and leaving room for some morning “me” time. 
+    This includes meditation, reading, self-care, and/or extracurriculars.</p>
+
```

Comment:
Delete the spaces between the ```<p>``` tags (there should be no line spaces) because this messes up the parser.